lstchain/datachecks/dl1_checker.py

```diff
@@ -335,6 +340,10 @@ def plot_datacheck(datacheck_filename, out_path=None, muons_dir=None):
 
     # aspect ratio of pdf pages:
     pagesize = [12., 7.5]
+    # at some point all the font sizes etc became messed up due to other
+    # updates in lstchain, hence the line below, which brings everything back
+    # to normal:
+    mpl.rc_file_defaults()
```

Comment:
@vuillaut is it possible that `ctaplot` is overwriting any matplotlib default? Not sure why we would have to call  `rc_file_defaults` to come back to the default settings otherwise...

Reply:
Hum... no, I don't think so 🤔 
Are you sure it's the case on any computer/install?

Reply:
@morcuended reported it while doing some tests in the IT cluster and @moralejo also got it after merging the latest master (not sure after which changes) on his Mac (?), but it was not there before...

Reply:
I just reproduced it with a fresh lstchain installation on my laptop. Also while running it here locally I realized that some plots pop-up. Would it be possible to avoid that and just produce the pdf file? I see in some places `cam.show()` which should not happen when running in batch mode without a graphical interface.

Reply:
Ok I looked into it. Possibily `ctaplot` actually, that uses `seaborn` style.
Understandably annoying in some cases, I will fix this. In the meantime, `mpl.rc_file_defaults()` can be called after imports.

Reply:
but as @moralejo mentioned, this may also break other plots (I did not have the time to test...), and the `mpl.rc_file_defaults()` solution will only work for the scripts in which it is included, right?

Reply:
if #402 is accepted, we do not need these lines here (I tested it and it works for the datacheck, should also work for the rest of the lstchain plots)